### PR TITLE
Remove `jinja2>=3.0` dependencies and release `gradio==3.13.2`

### PR DESCRIPTION
**Pull request by abidlabs**
It turns out that `jinja2.utils.htmlsafe_json_dumps` was introduced in 3.0, which means that on environments (like Colab) where it is not the default, it will break. Rather than pinning the jinja version, this manually implements the `htmlsafe_json_dumps` method.

Note that I've included `markupsafe` in the `requirements.txt` file but this is not really a new dependency as it is already a dependency of `jinja2`.

### CODE REVIEW

**Comment by abidlabs** [comment]
Both local and public Gradio apps seem to work on Colab with the latest changes to requirements.txt file: https://colab.research.google.com/drive/1wlc8vncpPLFOTSzU1tbO8pzmCb90yvri?usp=sharing

Should be good to release `gradio==3.13.2` now!

File: gradio/routes.py [hunk 1]
```diff
@@ -73,7 +73,13 @@ def _render_str(content: Any) -> str:
 
 
 def toorjson(value):
-    return htmlsafe_json_dumps(value, dumps=ORJSONResponse._render_str)
+    return markupsafe.Markup(
```
Comment (freddyaboulton) [inline gradio/routes.py:76]: Is this the implementation of html_safe_json_dumps?

**Review by freddyaboulton — APPROVED** [review]
Thanks @abidlabs 👏

File: gradio/routes.py (hunk 1 above)
Comment (abidlabs) [reply gradio/routes.py:76]: Yup basically

**Pull request merged**